agent.go

```diff
@@ -59,7 +59,7 @@ func Run(ctx context.Context, configFilePath string, hostDNSServer DNSServer,
 
 	config := &config{}
 	if err := config.init(configFilePath); err != nil {
-		writeStatus(fmt.Sprintf("Error reading config file %v", err))
+		_ = writeStatus(fmt.Sprintf("Error reading config file %v", err))
```

Comment:
@h0x0er you can remove the `_ =` part. That is redundant. Same applies to other places as well.